src/Events/Controllers/InboundController.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Threading.Tasks;
+
+using Altinn.Platform.Events.Models;
+using Altinn.Platform.Events.Services.Interfaces;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace Altinn.Platform.Events.Controllers
+{
+    /// <summary>
+    /// Controller responsible for pushing events to internal event queues.
+    /// </summary>
+    [Route("events/api/v1/inbound")]
+    [ApiController]
+    public class InboundController : ControllerBase
+    {
+        private readonly IInboundService _inboundService;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="InboundController"/> class.
+        /// </summary>
+        public InboundController(IInboundService inboundService)
+        {
+            _inboundService = inboundService;
+        }
+
+        /// <summary>
+        /// Post a previously registered cloudEvent to the inbound events queue
+        /// and further processing by the EventsInbound Azure function
+        /// </summary>
+        /// <returns>The application metadata object.</returns>
```

Comment:
Returns here must be updated

Reply:
There is a lot of code in this PR that appear to be new, but isn't. This is one example. I've allowed @b-brodie-digdir to focus on finishing up the actual focus of the PR and ignore old mistakes this time around.